arangod/RocksDBEngine/RocksDBEngine.cpp

```diff
@@ -2422,6 +2422,21 @@ bool RocksDBEngine::canUseRangeDeleteInWal() const {
   return ServerState::instance()->isSingleServer();
 }
 
+void RocksDBEngine::flushColumnFamilies(bool wait) {
+  // we are intentionally not flushing cfs "document" and "primary", as these
+  // are expected to be written to every now and then by statistics etc.
+  LOG_TOPIC(DEBUG, Logger::ENGINES) << "flushing column families";
+
+  rocksdb::FlushOptions options;
+  options.wait = wait;
+  
+  _db->Flush(options, RocksDBColumnFamily::definitions());
```

Comment:
does rocksdb flush CFs automatically when they are written to?

Reply:
I think if does so if the underlying memtables get full.
But for CFs that is hardly anything written to, this may not happen for a very long time.